### ISSUE TITLE
Foxx Services aren´t distributed to all coordinators in a cluster

### ISSUE THREAD

**Issue by MrPieces** [arangodb/arangodb#2083]
## my environment running ArangoDB

I'm using the latest ArangoDB of the respective release series:
- [ ] 2.8
- [x] 3.0
- [ ] self-compiled devel branch

On this operating system:
- [ ] DCOS on
  - [ ] AWS
  - [ ] Azure
  - [x] own infrastructure
- [ ] Linux 
  - [ ] Debian .deb
  - [ ] Ubuntu .deb
  - [ ] SUSE   .rpm
  - [ ] RedHat .rpm
  - [ ] Fedora .rpm
  - [ ] Gentoo
  - [ ] docker - official docker library
  - [ ] other:
- [ ] Windows, version: 
- [ ] MacOS, version:
### this is an AQL-related issue:

[ ] I'm using graph features
### Foxx

This question was originally asked by Martin Frstr in the GoogleGroup: https://groups.google.com/forum/#!topic/arangodb/h4PYiCetznc

I struggle to install Foxx-Apps via the Webinterface reliably. Even when the cluster is up and running with all nodes beeing sound and healthy, i can't figure out how i'm supposed to install a Service troughout the whole cluster.

I came across different scenarios so far: Once installing on one coordinato mirrored the service meta informations to the other coordinator. But on the second coordinator none of the sources was available and i could neither open the "API" Tab nor could i communicate with the service via the WEB-API. On the first coordinator everything worked perfectly.

In another installation attempt all of a sudden it worked on both, but i since than could not replicate that success.

So long story short - My Question: How is one supposed to install a Foxx-Service in a Cluster Setup with multiple Coordinators. (i'm under the Impression that i red something about it somewhere in the docs that stated you should install the service on every coordinator, but i can't find it anymore).

Any hint/help would be greatly appreciated!

Thanks in advance

**Comment by pluma** [comment]
Will be part of the next 3.2 release.

**Comment by dothebart** [comment]
We're pleased to anounce that ArangoDB 3.2 containing this is now available for download - Closing.
